packages/playground/lib/src/main.js

```diff
@@ -1,3 +1,6 @@
 export default function myLib(sel) {
+  // Force esbuild helpers
+  console.log({ ...'foo' })
```

Comment:
It might be a good idea to specify that this is the reason that the helper function is included. https://github.com/evanw/esbuild/issues/951